hw-model/c-binding/caliptra_model.h

```diff
@@ -26,9 +26,10 @@ enum {
   // more error enums here
 };
 
+
```

Comment:
Doesn't have to be in this PR, but what do you think of using [cbindgen](https://github.com/mozilla/cbindgen) to generate this file from lib.rs?

Kinda like [verilated/generate_rust_bindings.sh](https://github.com/chipsalliance/caliptra-sw/blob/main/hw-latest/verilated/generate_rust_bindings.sh) but going the other direction?

Reply:
done